src/cclib/io/moldenwriter.py

```diff
@@ -76,6 +76,21 @@ def _gto_from_ccdata(self):
 
         return block
 
+    def _scfconv_from_ccdata(self):
+        """Create [SCFCONV] section using gbasis."""
+
+        # scf-first    1 THROUGH   12
```

Comment:
So is there any reasons have some many spaces here?

Also, please cover the edge cases in the unit test.